### PR TITLE
dev-lisp/c2ffi: Bump version to c2ffi-13.0.0 (bug #840610)

### PR DESCRIPTION
**Pull request by hjudt**
Version bump to c2ffi-13.0.0.

Upstream bug: https://github.com/rpav/c2ffi/issues/106
Gentoo bug: https://bugs.gentoo.org/840610

### CODE REVIEW

File: dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild [hunk 1]
```diff
@@ -16,9 +16,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 IUSE=""
```
Comment (ConiKost) [inline dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild:17]: Im empty, drop it.

File: dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# Copyright 2019-2021 Gentoo Authors
+# Copyright 2019-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```
Comment (ConiKost) [inline dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild:4]: Why not EAPI=8?

**Review by ConiKost — CHANGES_REQUESTED** [review]

File: dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild (hunk 1 above)
Comment (hjudt) [reply dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild]: Done.

File: dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild [hunk 3]
```diff
@@ -16,9 +16,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 IUSE=""
 
-DEPEND="sys-devel/clang:12=
-	sys-devel/llvm:12=
-"
+DEPEND="sys-devel/clang:13="
 RDEPEND="${DEPEND}"
 
 S="${WORKDIR}/c2ffi-${MY_COMMIT}"
```
[the remark this answers is not in the document]
Comment (hjudt) [reply dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild]: Also fixed, should be in the right order now.

File: dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild [hunk 4]
```diff
@@ -1,24 +1,21 @@
...
+EAPI=8
 
 inherit cmake llvm
 
-MY_COMMIT="3a92064409d258e1688727266c3f023a502e8866"
+MY_COMMIT="bfa50485ffa86b886215c72ea1e43dbd3acaf940"
 
 DESCRIPTION="Clang-based FFI wrapper generator for Common Lisp"
 HOMEPAGE="https://github.com/rpav/c2ffi"
 SRC_URI="https://github.com/rpav/c2ffi/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (ConiKost) [inline dev-lisp/c2ffi/c2ffi-13.0.0.20220626.ebuild:13]: No new line here please.

**Review by ConiKost — CHANGES_REQUESTED** [review]
Just a small nit pick: Please Add ```Closes: https://bugs.gentoo.org/840610``` to your commit.

**Pull request closed without merging**